Makefile

```diff
@@ -193,6 +197,10 @@ pao-functests-only:
 	hack/show-cluster-version.sh
 	hack/run-functests.sh
 
+.PHONY: pao-functests-latency-testing
+pao-functests-latency-testing: dist-latency-tests
+	GINKGO_SUITS="./test/e2e/performanceprofile/functests/0_config ./test/e2e/performanceprofile/functests/5_latency_testing" ./hack/run-latency-testing.sh
```

Comment:
better do like that:
.PHONY: pao-functests-only
pao-functests-only:
	@echo "Cluster Version"
	hack/show-cluster-version.sh
	hack/run-functests.sh

and set GINKGO_SUITS as in this one:
https://github.com/openshift/cluster-node-tuning-operator/blob/977dd8f366a111ede42285fbe237905d979da30d/hack/run-functests.sh#L3

Reply:
updated, Thanks!